folder_paths.py

```diff
@@ -85,6 +89,28 @@ def get_directory_by_type(type_name):
         return get_input_directory()
     return None
 
+def filter_files_content_types(files: List[str], content_types: List[str]) -> List[str]:
```

Comment:
Can we make `content_types` enum?

Reply:
Yes absolutely, thanks for the feedback. The other mimetype types are "text", "application", "multipart", "font", "model", "message", and "x-type." Do you think I should I include them all in the type def since all would technically be possible?

Reply:
It is meant to be called with "image"/"video"/"audio" types, I think it is fine to keep the list short for now.